pkg/cli/project_create.go

```diff
@@ -23,11 +23,17 @@ acorn project create acorn.io/username/new-project
 		Args:              cobra.MinimumNArgs(1),
 		ValidArgsFunction: newCompletion(c.ClientFactory, projectsCompletion).complete,
 	})
+	// This will produce an error if the region flag doesn't exist or a completion function has already
+	// been registered for this flag. Not returning the error since neither of these is likely occur.
+	if err := cmd.RegisterFlagCompletionFunc("region", newCompletion(c.ClientFactory, regionsCompletion).complete); err != nil {
+		cmd.Printf("Error registering completion function for --region flag: %v\n", err)
+	}
 	return cmd
 }
 
 type ProjectCreate struct {
 	client ClientFactory
+	Region string `usage:"Region to create project in" short:"r"`
```

Comment:
Lets hold off on the short form and just have `--region` right now.  It's easier to add a short form later, but not remove it.

Reply:
I removed it from the run command and not here. I will remove here too. Thanks for catching it.